src/components/Draggable/index.js

```diff
@@ -51,7 +52,12 @@ class Draggable extends Component {
       const child = this.draggableDom.firstChild;
       const height = (child && child.clientHeight) || 0;
       const width = (child && child.clientWidth) || 0;
-      this._isClick = false;
+      if (
+        Math.abs(newPositionX - this._positionXOnMouseDown) > 1 ||
```

Comment:
Preferably modify 1 to something like this.props.threshold, and have the threshold default to maybe 5 to 15. Test out which number works better.
The idea behind a larger threshold number is that if we have users using touch screen and decides to use touch to move the badge, the touch sensor will mostly likely sense some movements because of the size of people's fingers. For now, maybe a 5 pixel threshold is good enough. Also, I personally have came across some users who just can't click without moving their mouse, so a reasonable threshold is ideal.